### ISSUE TITLE
Error npm run rename if ThemeName is written in CamelCaseNotation

### ISSUE THREAD

**Issue by MrAndreius** [justintadlock/mythic#73]
Fatal error: Uncaught Error: Class 'ThemeName\Providers\AppServiceProvider' not found in .......\wp-content\themes\themename\vendor\justintadlock\hybrid-core\src\Core\Application.php on line 187

To make it work, need to manually correct the class name in the files. Themename = > ThemeName
vendor\composer\autoload_static.php
vendor\composer\autoload_psr4.php

**Comment by justintadlock** [comment]
Can you cross-post this in the Theme Claim issue tracker, which is the package we're using? https://github.com/m-e-h/theme-claim

**Comment by lkraav** [comment]
Fairly certain duplicate of https://github.com/m-e-h/theme-claim/issues/6

**Comment by justintadlock** [comment]
Yep, that would be.  I remember coming across that now with an all-caps theme name.

**Comment by MrAndreius** [comment]
found another place where the theme name is written incorrectly
composer.json
`"autoload"    : {
		"psr-4" : {
			"Themename\\" : "app/" // need ThemeName
		}
	},`
this may then again cause a loading error

**Comment by m-e-h** [comment]
This should be good with https://github.com/m-e-h/theme-claim/commit/5dc43e39d4864c50d29510210ca34ca2695c26f4. :tada:  
Keep me posted if it's not.

Also, probably gonna need to update the `package.json` dep to 0.2.0
